src/actions/Packages/packageActions.js

```diff
@@ -49,6 +49,15 @@ export const uploadPackage = (packageInfo, uploader) => {
 	if (packageInfo.packageType === "Other") {
 		packageInfo.packageType = packageInfo.packageTypeOther;
 	}
+	let activeFiles = uploader.methods.getUploads({
+		status: [ qq.status.SUBMITTED, qq.status.PAUSED ]});
+	packageInfo.attachments = activeFiles.map((file) => {
+		return {
+			id: file.uuid,
```

Comment:
I don't think we should send the uuid from fineUploader...we are creating our own id on the back end.